### PR TITLE
Update dependencies and fix scroll error on console

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] Update dependencies
- [ ] Fix scroll error on console

### CODE REVIEW

File: frontend/app/package.json [hunk 1]
```diff
@@ -55,33 +55,33 @@
...
     "@vue/eslint-config-prettier": "7.0.0",
     "@vue/eslint-config-typescript": "11.0.0",
     "@vue/test-utils": "1.3.0",
     "c8": "7.11.3",
     "cross-env": "7.0.3",
     "dotenv": "14.3.0",
     "dotenv-cli": "5.1.0",
-    "electron": "20.2.0",
-    "electron-builder": "23.3.3",
-    "electron-notarize": "1.2.1",
```
Comment (kelsos) [inline frontend/app/package.json:83]: why was notarize removed?

Comment (kelsos) [reply frontend/app/package.json:83]: Ok package name change

File: frontend/app/package.json [hunk 2]
```diff
@@ -98,15 +98,15 @@
     "stylelint-config-recommended-scss": "4.3.0",
     "stylelint-order": "4.1.0",
     "stylelint-scss": "3.21.0",
-    "ts-loader": "9.2.8",
-    "typescript": "4.7.4",
-    "unplugin-auto-import": "0.11.2",
-    "unplugin-vue-components": "0.22.7",
+    "ts-loader": "9.4.1",
+    "typescript": "4.8.4",
```
Comment (kelsos) [inline frontend/app/package.json:102]: do we get errors from the eslint-typecript?

File: frontend/app/package.json [hunk 3]
```diff
@@ -55,33 +55,33 @@
     "vue-i18n": "8.27.0",
```
Comment (kelsos) [inline frontend/app/package.json:55]: Was the new minor version of vue causing issues?

Comment (lukicenturi) [reply frontend/app/package.json:55]: I'm pretty sure I tried it, and it was broken.

Comment (kelsos) [reply frontend/app/package.json:55]: ok

Comment (kelsos) [reply frontend/app/package.json]: It seems that it gives to many errors, which is interesting. I wonder which change causes it. Updating vue-tsc doesn't make a difference either.

File: frontend/app/package.json [hunk 4]
```diff
@@ -98,15 +98,15 @@
     "stylelint-config-recommended-scss": "4.3.0",
     "stylelint-order": "4.1.0",
     "stylelint-scss": "3.21.0",
-    "ts-loader": "9.2.8",
-    "typescript": "4.7.4",
-    "unplugin-auto-import": "0.11.2",
-    "unplugin-vue-components": "0.22.7",
+    "ts-loader": "9.4.1",
```
Comment (kelsos) [inline frontend/app/package.json:101]: I have been wondering if we do actively use ts-loader anywhere or if it is a leftover from the webpack times.

**Review by kelsos — APPROVED** [review]
Ok I did a quick test, and nothing seems completely broken, and the scroll fix seems to work. 

lgtm @lukicenturi

**Pull request merged**